src/Plugin/WebformElement/CivicrmContact.php

```diff
@@ -488,7 +488,7 @@ public static function wf_crm_fill_contact_value(WebformInterface $node, array $
     $cid = wf_crm_aval($element, '#default_value', '');
     $contactComponent = \Drupal::service('webform_civicrm.contact_component');
     if ($element['#type'] == 'hidden') {
-      if (!$component['#show_hidden_contact']) {
+      if (!isset($component['#show_hidden_contact']) || !$component['#show_hidden_contact']) {
```

Comment:
This means something else is funky with setting up the form, but it removes the error.